lib/parameters.py

```diff
@@ -181,6 +181,30 @@ def gen_avc_vbr_la_parameters(spec, profiles):
   params = gen_avc_vbr_la_variants(spec, profiles)
   return keys, params
 
+def gen_avc_forced_idr_variants(spec, profiles, tested):
+  for case, params in spec.items():
+    for variant in copy.deepcopy(params.get("forced_idr", [])):
+      uprofile = variant.get("profile", None)
+      cprofiles = [uprofile] if uprofile else profiles
+      rcmode = variant.get("rcmode", None)
```

Comment:
This implies that `rcmode` might not be required, yet it is unconditionally indexed on line 195.  AFAICT, `rcmode` is required, thus we should use `rcmode = variant["rcmode"]` here, too.  If user config forgets to set it, then it is a config/spec violation and will throw exception.  This is desired because it forces the user to specify it in order for it to be a valid config.  Also, this will allow us to use `rcmode` variable directly on line 195 instead of doing another lookup.